modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomViews.testcase

```diff
@@ -386,6 +386,65 @@ definition {
 		}
 	}
 
+	@description = "LPS-170529 - Verify if it is possible to create a filter by relationship columns made from the system objects to custom objects"
+	@priority = "5"
+	test CanCreateFilterByRelationshipColumnFromSystemObject {
+		task ("Given a system object related to another custom object, in a One-to-Many relationship") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 170529",
+				objectName = "CustomObject170529",
+				pluralLabelName = "Custom Objects 170529");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject170529");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "User",
+				objectName_2 = "CustomObject170529",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject170529");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "Custom Field 170529",
+				objectName = "CustomObject170529",
+				value = "Entry Test");
+		}
+
+		task ("When the user creates a filter and the relationship column can be selected as the filter") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 170529");
+
+			ObjectCustomViews.goToViewsTab();
+
+			ObjectCustomViews.addObjectViewViaUI(viewName = "Custom Views");
+
+			ObjectCustomViews.goToViewsDetails(label = "Custom Views");
+
+			ObjectCustomViews.markViewAsDefault();
+
+			ObjectCustomViews.goToViewsBuilderTab();
+
+			ObjectCustomViews.addColumnsViaUI(addColumns = "Relationship");
+
+			PortletEntry.save();
+		}
+
+		task ("Then the user can filter relationship entries") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject170529");
+
+			ObjectPortlet.assertEntryColumnPresent(column = "Relationship");
```

Comment:
This test would be more effective if you assert specific entries are being filtered rather than asserting that the option is available.